static/src/javascripts/projects/common/modules/analytics/google.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+
+const trackerName = config.googleAnalytics.trackers.editorial;
+const send = `${trackerName}.send`;
+
+const extractLinkText = (el: ?HTMLElement): ?string => {
```

Comment:
i guess because `querySelector` can be? maybe we need a wrapper around it to just return an array, nothing more fancy, and then we should always expect to operate on an array of elements (which could be zero length)?